test-utils/testlib/src/alphanet_utils.rs

```diff
@@ -66,7 +67,9 @@ impl NodeConfig {
     }
 }
 
-pub trait Node {
+pub trait Node: Send + Sync {
+    fn account_id(&self) -> Option<&AccountId>;
```

Comment:
Why do we make this optional?

Reply:
because you can run node without account now (e.g. if you are not an authority)